woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -107,6 +108,23 @@ public function is_available()
 	}
 
 	/**
+	* Detects if the page only accepts the selected gateways.
+	*
+	* @return boolean
+	*/
+	public function is_gateway()
```

Comment:
Function is nice and good but this method name is somewhat misleading... :/
`is_current_order_gateway` maybe